index.js

```diff
@@ -129,6 +129,9 @@ module.exports = function(content) {
 	}
 
 	if(config.interpolate && config.interpolate !== 'require') {
+		// Double escape quotes so that they are not unescaped completely in the template string
+		content = content.replace(/\\"/g, "\\\\\"");
```

Comment:
`\\\\\('|")` (5) => `\\\\('|")` (4) ?

Reply:
I'm not sure I understand.
The two regexes could be combined into one `.replace(/\\("|')/g, "\\\\$1")` or what do you mean?